config/default/core.entity_form_display.node.article.default.yml

```diff
@@ -113,7 +114,9 @@ content:
     type: metatag_firehose
     weight: 6
     region: content
-    settings: {  }
+    settings:
```

Comment:
What does this new addition add?  Just curious.

Reply:
this was just a re-export of default. 🤷